### ISSUE TITLE
Exclude Kane's new Therapedic bedding line from promotions

### ISSUE THREAD

**Issue by kbower** [kbower/tickettest1#1555]
_Imported from trac ticket 1555.
Created by **wayne, kbower**
Opened in trac: Mar 26, 2012, 9:50:19 AM
Last modified in trac: Mar 27, 2012, 1:19:40 PM_
***
_by wayne, kbower:_
Kent,
 
We have added a new bedding line to replace the King Koil brand on our floors.
 
The new vendor is Therapedic. We are offering a TV Bed promotion in this series as well, and it should be excluded from all promotions, as was the Grand TV bed in the past.
 
The new bed is called the Splendor. Here are the SKU's that need to be excluded.  Vend: THER    Family: 70009

         Package SKU

QN      SPLEN01        506700093 matt           508700110 foundation
KG      SPLEN02        509700096 matt           511700115 foundation
QN      SPLEN03        506700093 matt           507700119 Low Profile foundation
KG      SPLEN04        509700096 matt           510700114 Low Profile foundation

Let me know what questions you may have.

Thanks,

Tim

**Comment by kbower** [comment]
_**kbower** trac comment on Mar 26, 2012, 9:50:33 AM:_

Hi Tim,
    You said "We are offering a TV Bed promotion in this series as well, and it should be excluded from all promotions, as was the Grand TV bed in the past." 

I don't know the Grand TV bed by that name, so I don't know what you are referring to. 
Are you referring to vendor == 'KING' and family == '58455'?

When you said "it" should be excluded, are talking about excluding every Vendor 'THER' Family 70009 from every promotion?

Also, for purposes of bedding express shipping, I recall we needed to add the TVs to a list in the rule so that even if they get the TV, they still qualify for the $39.99 shipping.  Is this the case for the Therapedic line?  Have the TV SKUs remained the same ('MIDHDTV', 'MIDHDTV50', 'MIDHDTVUP'), or do we need to add to that list of TVs?

Kent

**Comment by kbower** [comment]
_**kbower** trac comment on Mar 26, 2012, 9:54:57 AM:_

```
SQL> select productid, regular from products where vendor = 'THER' and family = '70009';

PRODUCTID    REGULAR
--------- ----------
506700093     749.99
509700096     974.99

SQL> select productid, regular from products  where vendor = 'KING' and family = '58455';

PRODUCTID    REGULAR
--------- ----------
506584552     749.99
509584555     974.99
508584554          0
511584559          0
507584553          0
510584558          0

6 rows selected.
```
Why aren't the foundations in 'THER' '70009'?

**Comment by kbower** [comment]
_**kbower** trac comment on Mar 26, 2012, 5:32:35 PM:_

Kent,
 
You are correct. The KING vendor and family 58455 were the former Grand "TV Beds". 
 
"It" is the new Line of TV Beds, Vendor THER and family 70009. (The beds in the THER line qualify for our promotions and are not excluded.)
 
The TV SKU's have not changed from the previous promotions and will qualify for express delivery when sold with one of these mattress sets.
 
Let me know what other details you need.
 
Thanks,
Tim

**Comment by kbower** [comment]
_**kbower** trac comment on Mar 26, 2012, 5:39:08 PM:_

Talked to Tim:

We can assume that one of the four package SKUs was added:

QN SPLEN01 506700093 matt 508700110 foundation
KG SPLEN02 509700096 matt 511700115 foundation
QN SPLEN03 506700093 matt 507700119 Low Profile foundation
KG SPLEN04 509700096 matt 510700114 Low Profile foundation

(If they manually try to build the package then an exception report will catch that and they'll be charged apparently, so go with the assumption they added the package SKU)

**Comment by kbower** [comment]
_**kbower** trac comment on Mar 27, 2012, 1:19:40 PM:_

(Changeset [[2487]](../commit/94bd3b63e5a622eda876e4e870d4dce198b1cab8))
 * Exclude Kane's new Therapedic bedding line from promotions (closes #1555)

**Comment by kbower** [comment]
_**kbower** trac status change on Mar 27, 2012, 1:19:40 PM:_ **closed**
